src/Screens/Main/Settings/Password.tsx

```diff
@@ -46,21 +46,34 @@ export default ({ navigation }: Props) => {
     repeatedNewPassword.length > 0 &&
     newPassword === repeatedNewPassword;
 
+  const onButtonPress = () => {
+    dispatch(state.changePassword({ currentPassword, newPassword }));
+  };
+
   const onGoBack = () => {
     navigation.goBack();
   };
 
-  const onButtonPress = () => {
-    dispatch(state.changePassword({ currentPassword, newPassword }));
+  const resetAndGoBack = () => {
+    setCurrentPassword('');
+    setNewPassword('');
+    setRepeatedNewPassword('');
+    dispatch(state.reset);
+    onGoBack();
   };
+
   const requestState = useSelector(state.select);
 
+  const changeIsResolved =
```

Comment:
Name wise isRequestResolved or isChangeRequest resolved would be better. If u start with verb, readers thinks this is a function

Reply:
And good habit to start boolean vars with is/has etc.